Revise index generation to use new content tree (Issue 2418 task 8)

alflennik opened the pull request:
See https://github.com/w3c/aria-practices/issues/2418 for more information about the task.

mcking65 reviewed (changes requested):
Tested, working great!

However, I think we need to do two more things with this PR.

1. Delete the content/index.html file generated by the old version of the script.
2. Change content of scripts/reference-tables.template to match what is in prod on the WAI site. One of our goals of the restructure is for all content in prod to be in the aria-practices repo. The jekyll build should not add or modify content.

alflennik commented:
@mcking65 Thanks for the feedback, and I implemented the fixes. I had indeed missed the previous index.html file, thanks for noticing it. I also updated the copy to match the production site.

mcking65 commented on scripts/reference-tables.template:
```diff
@@ -11,15 +11,12 @@
   <script src="js/app.js"></script>
```
The paths on the file refs in the script and css don't appear correct

mcking65 reviewed (changes requested):
Looking good; just one more thing. I think we need some edits to the head in the template. I don't know which script and css is needed. We should think about both what is needed in prod and what helps people when viewing the content locally in clones of the repo.

alflennik commented:
@mcking65 Thanks again for pointing out the broken link, I fixed it and several other broken links on this page, and now the JS and CSS running on this page are the same as the other example pages.
